packages/iTwinUI-react/src/core/Table/Table.test.tsx

```diff
@@ -3932,21 +3655,16 @@ it('should add expander column manually', () => {
   );
   const isRowDisabled = (rowData: TestDataType) => rowData.name === 'Name2';
   const columns: Column<TestDataType>[] = [
+    ExpanderColumn({ subComponent, isDisabled: isRowDisabled }),
     {
-      Header: 'Table',
-      columns: [
-        ExpanderColumn({ subComponent, isDisabled: isRowDisabled }),
-        {
-          id: 'name',
-          Header: 'Name',
-          accessor: 'name',
-        },
-        {
-          id: 'description',
-          Header: 'Description',
-          accessor: 'description',
-        },
-      ],
+      id: 'name',
+      Header: 'Name',
+      accessor: 'name',
+    },
+    {
+      id: 'description',
+      Header: 'Description',
+      accessor: 'description',
     },
   ];
   const { container } = renderComponent({
```

Comment:
Lines 3698 and 3732 need to be updated to remove the `Header: 'Table'` thing

Reply:
Removed